packages/next/src/server/web/spec-extension/revalidate-path.ts

```diff
@@ -1,11 +1,26 @@
 import { revalidateTag } from './revalidate-tag'
-import { NEXT_CACHE_IMPLICIT_TAG_ID } from '../../../lib/constants'
+import { isDynamicRoute } from '../../../shared/lib/router/utils'
+import {
+  NEXT_CACHE_IMPLICIT_TAG_ID,
+  NEXT_CACHE_SOFT_TAG_MAX_LENGTH,
+} from '../../../lib/constants'
 
-export function revalidatePath(path: string, type?: 'layout' | 'page') {
-  path = `${NEXT_CACHE_IMPLICIT_TAG_ID}${path}`
+export function revalidatePath(originalPath: string, type?: 'layout' | 'page') {
+  if (originalPath.length > NEXT_CACHE_SOFT_TAG_MAX_LENGTH) {
+    console.warn(
+      `Warning: revalidatePath received "${originalPath}" which exceeded max length of ${NEXT_CACHE_SOFT_TAG_MAX_LENGTH}.`
```

Comment:
Should this link to docs?

Reply:
Added in https://github.com/vercel/next.js/pull/55083/commits/03f11084918b9d0bf9d2a8f7b18cdef1cc93f293